Cargo.toml

```diff
@@ -19,12 +19,11 @@ is-it-maintained-issue-resolution = { repository = "DarkEld3r/os_info" }
 is-it-maintained-open-issues = { repository = "DarkEld3r/os_info" }
 
 [features]
-default = ["serde", "serde_derive"]
+default = ["serde"]
 
 [dependencies]
 log = "0.4.5"
-serde = { version = "1.0.0", optional = true }
-serde_derive = { version = "1.0.0", optional = true }
+serde = { version = "1.0.0", features = ["derive"], optional = true }
```

Comment:
Ah yes, this is something I wondered on my pull request. Technically, someone could incorrectly choose only the `serde` dependency but not `serde_derive` or vice-versa. I tried to create a feature including both dependencies (so you either have this parent one or not), but I couldn't create a feature named the same as a dependency. I didn't know `serde` had its own features, but this is a nice solution to that.